playbook/app/pb_kits/playbook/pb_bread_crumbs/docs/_bread_crumbs_default.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import BreadCrumbs from '../_bread_crumbs.jsx'
+import { Icon, Title } from '../../';
+import BreadCrumbItem from '../_bread_crumb_item'
+
+const Link = (props) => <BreadCrumbItem {...props} />
+const BreadCrumbsDefault = (props) => {
+
+  return (
+      <BreadCrumbs
+          text="+1"
+          {...props}
+      >
+        <Icon icon="home" size="1x" />
+        <BreadCrumbItem {...props} href="/home">
+          <Title size="4" text="Home" />
+        </BreadCrumbItem>
+          <Icon icon="users" size="1x" />
+        <Link {...props} href="/users">
+          <Title size="4" text="Users" />
+        </Link>
+          <Icon icon="user" size="1x" />
+        <Link {...props} >
+          <Title size="4" text="User" />
```

Comment:
```suggestion
          <Title size="4" tag="span" text="Home" />
        </BreadCrumbItem>
          <Icon icon="users" size="1x" />
        <Link {...props} href="/users">
          <Title size="4" tag="span" text="Users" />
        </Link>
          <Icon icon="user" size="1x" />
        <Link {...props} >
          <Title size="4" tag="span" text="User" />
```

Reply:
For the examples, while the ticket mentions using a "Title", I think that actually using the `<Title>` HTML landmark works against accessibility efforts and instead would suggest simply using a `<span>` by sending the prop.